### DISCUSSION TITLE
Synchronization with 360° photos.

### DISCUSSION THREAD

**Discussion by danielmga** [opengisch/QField#4887] [Q&A]
I need to carry out a field project that requires the visualization of 360° photos in QField. Does anyone know if it's possible to do this?

**Comment by ZsanettMed** [comment]
Hi @danielmga!

Unfortunately, the visualization of 360° photos in QField is not currently possible. However, I've posted your idea on our [QField Idea Platform](https://ideas.qfield.org/app-feature-requests/p/visualization-of-360-photos) where you and others can upvote it. This will help us prioritize new features based on user demand.

Best regards!
Zsanett
